sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -57,6 +57,18 @@
         PremiumPageBlobTier)
 
 
+def _get_blob_name(blob):
```

Comment:
should we rename this to make it generic to accept container too...

Reply:
we can refactor this later, I really wish to keep this last minute PR focused on point. It's private API anyway, we have full control.